content/applications/finance/payment_acquirers/authorize.rst

```diff
@@ -115,6 +115,91 @@ order is canceled, you can click on *Void Transaction* to unlock the funds from
 .. note::
    With other payment acquirers, you can manage the capture in their own interfaces, not from Odoo.
 
+Authorize.Net statement export
```

Comment:
As explained further down, customers may use different software (Excel/G. Sheets/Spreadsheet), the UI element may change from software to software, so I prefer to bolden the **concept** rather than the :guilabel:`UI element`

In 14.0 the journal is "Authorize.Net", in 15.0 and beyond, the journal longer exists (by default at least), and is just "bank"